packages/teams-js/test/public/media.spec.ts

```diff
@@ -745,8 +742,6 @@ describe('media', () => {
       });
 
       it('videoController stop function returns SdkError to callback when parent rejects message"', async () => {
```

Comment:
I don't think this test is still testing what the title says it is. Title says we're checking that SdkError gets passed to callback, but test is testing whether we catch the error when calling.

Are there tests testing the callback still? On a quick scan I didn't see any.